cvat-ui/src/actions/auth.actions.ts

```diff
@@ -17,13 +17,15 @@ export const loginError = (error = {}) => (dispatch: any) => {
   });
 }
 
-export const loginAsync = (username: string, password: string) => {
+export const loginAsync = (username: string, password: string, history: any) => {
   return (dispatch: any) => {
     dispatch(login());
 
     return (window as any).cvat.server.login(username, password).then(
       (authenticated: any) => {
+        localStorage.setItem('session', 'true');
```

Comment:
We use localStorage to define whether we would be redirected? I am not sure it's a good idea.